modules/appeals_api/config/schemas/10182_headers.json

```diff
@@ -64,6 +66,7 @@
     "nodCreateHeadersBirthDate": {
       "allOf": [
         { "description": "birth date" },
+        { "maxLength": 10 },
```

Comment:
Same here. I feel like the date definition itself should have the `minLength` & `maxLength` associated with it since it knows the number of characters it requires for a valid date.

Reply:
Wouldn't the minLength be the same as maxLength in the case of date?